cloudify_cli/execution_events_fetcher.py

```diff
@@ -72,8 +64,29 @@ def _fetch_events(self):
             raise
         return events
 
-    def fetch_all(self):
-        return self.fetch_events(get_remaining_events=True)
+    def fetch_and_process_events(self, events_handler=None):
+        total_events_count = 0
+        deadline = time.time() + self._timeout
+
+        while True:
+            if time.time() > deadline:
+                raise EventProcessingTimeoutError(
+                    self._execution_id,
+                    'events/log fetching timed out for execution timed out')
```

Comment:
this message is a little confusing
'events/log fetching timed out for execution timed out'